src/Hello.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Hello = function(props) {
```

Comment:
Looks good Alex! Do you think it's possible to use the es6 function declaration syntax here?